src/stylesheets/main.css

```diff
@@ -382,6 +382,14 @@ p.important {
   .rhombus-1 {
     clip-path: polygon(0 3%, 100% 0, 96% 98%, 3% 100%);
   }
+
+  .rhombus-2 {
+    clip-path: polygon(8% 10%, 91% 0, 100% 100%, 0 100%);

```

Comment:
Changing `91% 0` to `91% 2.5%` will better align the two top-most corners:

**Before**:

<img width="1512" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/86cc31e1-5cef-44ef-8fba-be015a63103b">


**After**:
<img width="1512" alt="image" src="https://github.com/cal-itp/calitp.org/assets/3673236/a0f7e87d-17d2-43eb-b2ec-b7beb99a548e">